SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1270,7 +1272,10 @@ void USpatialReceiver::ResolvePendingOperations(UObject* Object, const FUnrealOb
 
 void USpatialReceiver::OnDisconnect(Worker_DisconnectOp& Op)
 {
-	NetDriver->HandleOnDisconnected(UTF8_TO_TCHAR(Op.reason));
+	if (GEngine != nullptr)
```

Comment:
When does this happen?

Reply:
UnrealEngine.cpp states it could be null so check before using. I agree we aren't consistent with checking this in our codebase however.

Reply:
Yeah I've no idea when this happens in practice - i've never seen it either but am going off the comment Mike mentioned